treeherder/push_health/usage.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+from treeherder.config import settings
+from treeherder.model.models import Push
+from treeherder.utils.http import make_request
+from treeherder.webapp.api.serializers import PushSerializer
+
+logger = logging.getLogger(__name__)
+
+
+def get_peak(facet):
+    peak = 0
+    date = 0
+    for item in facet['timeSeries']:
+        max = item['results'][-1]['max']
+        if item['inspectedCount'] > 0 and max > peak:
+            peak = max
+            date = item['endTimeSeconds']
+
+    return {'needInvestigation': peak, 'time': date}
+
+
+def get_latest(facet):
+    for item in reversed(facet['timeSeries']):
+        if item['inspectedCount'] > 0:
+            latest = item['results'][-1]
+            return {
+                'needInvestigation': latest['max'],
+                'time': item['endTimeSeconds']
+            }
+
+
+def get_usage(api_key=None):
+    nr_api_key = api_key or settings.NEW_RELIC_INSIGHTS_API_KEY
+    nrql = "SELECT%20max(needInvestigation)%20FROM%20push_health_need_investigation%20FACET%20revision%20SINCE%201%20DAY%20AGO%20TIMESERIES%20where%20repo%3D'{}'%20AND%20appName%3D'{}'".format(
+        'try', 'treeherder-prod')
+    new_relic_url = '{}?nrql={}'.format(settings.NEW_RELIC_INSIGHTS_API_URL, nrql)
+    headers = {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json',
+        'X-Query-Key': nr_api_key,
+    }
+    if not nr_api_key:
+        exception = EnvironmentError('NEW_RELIC_INSIGHTS_API_KEY not set on server.')
+        logger.exception(exception)
+        raise exception
+
+    resp = make_request(new_relic_url, headers=headers)
+    data = resp.json()
+    push_revisions = [facet['name'] for facet in data['facets']]
+    print(push_revisions)
```

Comment:
Did you want to keep the print instead of logging?

Reply:
Thanks.  Removed.  :)